jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/URLResourceFactory.java

```diff
@@ -103,7 +103,7 @@ public Resource newResource(final URI uri)
         }
     }
 
-    private static class URLResource extends Resource
+    public static class URLResource extends Resource
```

Comment:
EEK!
Please keep this private.
We don't want people using this class directly.

Reply:
That was accidental; reverted.